config/common/common.go

```diff
@@ -95,3 +97,18 @@ func TranslateReportPaths(r *report.Report, ts translate.TranslationSet) {
 		r.Entries[i].Context = context
 	}
 }
+
+// ValidateTranslated validates the final translated JSON config.
+func ValidateTranslated(r *report.Report, translated interface{}, contextTree tree.Node, ts translate.TranslationSet) {
```

Comment:
The general pattern seems to be to return a `Report` and have the caller merge it.

We're open-coding the rest of `TranslateBytes` within each config version, so it seems a bit odd to abstract out this specific part.

Reply:
I pushed a rebased version, but I'm not sure if your last sentence meant you'd prefer to get rid of this function and just have these few lines duplicated in the two callers.

Reply:
Without a larger refactoring, I think I'd be in favor of duplicating the lines in the callers.  But I won't block on it.